### PR TITLE
Test against nc 23

### PR DESCRIPTION
**Pull request by GretaD**
thats all i need to change, right?

### CODE REVIEW

File: .github/workflows/static-analysis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       runs-on: ubuntu-latest
       strategy:
           matrix:
-              ocp-version: [ 'dev-master', 'v20.0.4' ]
+              ocp-version: [ 'v22.0.0', 'v20.0.4' ]
```
Comment (ChristophWurst) [inline .github/workflows/static-analysis.yml:15]: let's leave dev-master and *add* v22.0.0-RC1

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         php-versions: [7.3, 7.4]
-        nextcloud-versions: ['stable20', 'stable21', 'master']
+        nextcloud-versions: ['stable20', 'stable21', 'stable22']
```
Comment (ChristophWurst) [inline .github/workflows/test.yml:16]: we should test both stable22 and master

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -59,7 +59,7 @@ jobs:
       strategy:
           matrix:
               php-versions: [7.4, 8.0]
-              nextcloud-versions: ['master']
+              nextcloud-versions: ['stable22']
```
Comment (ChristophWurst) [inline .github/workflows/test.yml:62]: this can stay on master

**Comment by ChristophWurst** [comment]
info.xml doesn't allow 23 yet

**Comment by GretaD** [comment]
> info.xml doesn't allow 23 yet

because i need to change this: https://github.com/nextcloud/mail/blob/master/appinfo/info.xml#L37
to 23, right?

**Comment by ChristophWurst** [comment]
> because i need to change this: https://github.com/nextcloud/mail/blob/master/appinfo/info.xml#L37
> to 23, right?

yes. That will bite us as soon as they've updated https://github.com/nextcloud/server/blob/4e9727037220a8acddf7a3478a047c0bc6fc4bae/version.php#L33 to say 23

**Comment by GretaD** [comment]
i see Joas pushed #5232  and #5230

**Comment by ChristophWurst** [comment]
> i see Joas pushed #5232 and #5230

Rebase to see what is left. I think your changes on the CI matrix are still required :)

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**